new-log-viewer/src/typings/config.ts

```diff
@@ -0,0 +1,34 @@
+enum CONFIG_NAME {
+    DECODER_OPTIONS = "decoderOptions",
+    THEME = "theme",
+    PAGE_SIZE = "pageSize",
+}
+
+enum LOCAL_STORAGE_KEY {
+    DECODER_OPTIONS_FORMAT_STRING = `${CONFIG_NAME.DECODER_OPTIONS}/formatString`,
+    DECODER_OPTIONS_LOG_LEVEL_KEY = `${CONFIG_NAME.DECODER_OPTIONS}/logLevelKey`,
+    DECODER_OPTIONS_TIMESTAMP_KEY = `${CONFIG_NAME.DECODER_OPTIONS}/timestampKey`,
+    THEME = CONFIG_NAME.THEME,
+    PAGE_SIZE = CONFIG_NAME.PAGE_SIZE,
+}
+
+type ConfigMap = {
+    [CONFIG_NAME.DECODER_OPTIONS]: {
+        formatString: string,
+        logLevelKey: string,
+        timestampKey: string,
+    },
+    [CONFIG_NAME.THEME]: string,
+    [CONFIG_NAME.PAGE_SIZE]: number,
+}
+
+type ConfigUpdate = {
```

Comment:
Shall we rename this as `ConfigUpdateType` to be consistent with the type naming in the other typings files?